src/haconiwa/tool/parallel_dev.py

```
@@ -0,0 +1,462 @@
+        return summary_file
+
+
+# Global manager instance
+manager = ParallelDevManager()
+
+
+@parallel_dev_app.command("claude")
+def claude_parallel(
+    files: Optional[str] = typer.Option(None, "-f", "--files", help="Comma-separated file paths"),
+    prompts: Optional[str] = typer.Option(None, "-p", "--prompts", help="Comma-separated prompts"),
+    file_list: Optional[Path] = typer.Option(None, "--file-list", help="File containing paths (one per line)"),
```

Ruff (0.11.9)

Do not perform function call `typer.Option` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)